pipelines/docker-build-oci-ta/patch.yaml

```
@@ -108,10 +108,15 @@
 - op: add
   # In the docker-build Pipeline, the snyk Task does not receive any parameters, so we cannot
   # append to it.
-  path: /spec/tasks/8/params
+  path: /spec/tasks/8/params/-
   value:
     - name: SOURCE_ARTIFACT
       value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
+- op: add
+  path: /spec/tasks/8/params/-
+  value:
+    - name: CACHI2_ARTIFACT
+      value: $(tasks.prefetch-dependencies.results.CACHI2_ARTIFACT)
```

```suggestion
    name: SOURCE_ARTIFACT
    value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
- op: add
  path: /spec/tasks/8/params/-
  value:
    name: CACHI2_ARTIFACT
    value: $(tasks.prefetch-dependencies.results.CACHI2_ARTIFACT)
```
I think the current state is causing the `params` section to contain nested lists

---

Thanks, yes this was causing double `-` for both of those params